libfwupdplugin/fu-pefile-firmware.c

```diff
@@ -242,6 +242,34 @@ fu_pefile_firmware_parse(FuFirmware *firmware,
 				      0x0,
 				      offset + FU_STRUCT_PE_COFF_OPTIONAL_HEADER64_OFFSET_CHECKSUM);
 
+	guint16 subsystem = 0;
```

Comment:
you can't mix declarations, but since it's better to add this to `fu_pefile_firmware_export` instead anyway, you should add the subsystem to `FuPefileFirmwarePrivate`